circle.yml

```diff
@@ -126,4 +124,8 @@ test:
       unicode
       unicode/utf16
       unicode/utf8
+    - >
+      gopherjs test --short --minify --run='^Test'
+      net/http/cookiejar
+      reflect
```

Comment:
It [looks like](https://circleci.com/gh/gopherjs/gopherjs/1062) only two examples are failing:

- `ExampleNew`, because it tries to perform network operations that GopherJS doesn't support (i.e., it tries to create a new HTTP server with `httptest.NewServer`).
- `ExampleStructOf` is failing because `reflect.StructOf` is not supported (see #499).

In the case of `net/http/cookiejar`, `ExampleNew` is the only example it has.

However, for `reflect`, there are [other examples](https://godoc.org/reflect#pkg-examples). Do you know if they work?

I think it'd be nice to simply disable the failing/unsupported examples, the way we already for tests, instead of outright disabling _all_ examples for the packages that contain examples that don't work. That'd be cleaner, and hopefully some examples in reflect packages do pass (and might catch regressions in the future).